packages/web-react/src/types/accordion.ts

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { ChildrenProps, StyleProps } from './shared';
```

Comment:
You are missing `TransferProps` for every component.

Reply:
done